### DISCUSSION TITLE
Creating user certificate for PDF signing

### DISCUSSION THREAD

**Discussion by registerme1963** [smallstep/certificates#659] [Q&A]
Hi,
Can somebody tell step-by-step procedure in creating PDF signing certificate for my organization users.  Can I create such certificates in bulk?
thanks
ps: sorry for this silly questions as I'm learning these stuffs

**Comment by tashian** [comment]
You'll need a certificate template for a document signing certificate, and you'll need a provisioner that can issue PDF signing certificates to your users. You could issue a lot at once, but it's much more secure if you use short lived certificates. The OIDC provisioner type will let your users get their own certificate on-demand from your step-ca, using SSO with your identity provider.

See [this blog post](https://smallstep.com/blog/x509-certificate-flexibility/) for an example certificate template for Adobe Reader. And see our [OIDC provisioner docs](https://smallstep.com/docs/step-ca/provisioners#oauthoidc-single-sign-on) for help setting that up.

Hope this helps.
